mycore-restapi/src/main/java/org/mycore/restapi/v2/access/MCRRestAPIACLPermission.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * This file is part of ***  M y C o R e  ***
+ * See http://www.mycore.de/ for details.
+ *
+ * MyCoRe is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MyCoRe is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MyCoRe.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.mycore.restapi.v2.access;
+
+import static org.mycore.access.MCRAccessManager.PERMISSION_READ;
```

Comment:
I don't like static imports, because you do not see where the values come from;
better:
public enum MCRRestAPIACLPermission {
    READ(MCRAccessManager.PERMISSION_READ), WRITE(MCRAccessManager.PERMISSION_WRITE), DELETE(MCRAccessManager.PERMISSION_DELETE);